module/dvfs/src/mod_dvfs.c

```diff
@@ -323,6 +329,7 @@ static int dvfs_handle_pending_request(struct mod_dvfs_domain_ctx *ctx)
     }
 
     if (ctx->config->retry_ms > 0) {
```

Comment:
Using the timer is conditional. If you don't need it make sure to set retry_ms=0 in the configuration.